src/telegram/process_update_task.rs

```diff
@@ -64,31 +77,59 @@ impl ProcessUpdateTask {
         if let UpdateContent::Message(message) = &self.update.content {
             let (chat_id, user_id, user) = Self::get_info_from_message(message);
 
-            Self::send_typing(message, &api).await?;
+            let params = Params {
```

Comment:
I think you can use TypedBuilder to create this struct

Reply:
Done :D [Typebuilder Params](https://github.com/pxp9/weather_bot_rust/pull/12/commits/077a11ba46144232bb1d19e56adcba1cb4baec00)